### PR TITLE
Update new modules (pricelists, users, products)

### PR DESCRIPTION
**Pull request by qgerome**
I don't have access to Lokalise so I was unable to upload updated strings.

I've also updated the documentation to use another styling for term definitions.

### CODE REVIEW

File: conf.py [hunk 1]
```diff
@@ -203,9 +200,3 @@ def setup(app):
 
 locale_dirs = ['locale/']   # path is example but recommended.
 gettext_compact = False     # optional.
-
-
-## Gen menu
-import gensidebar
```
Comment (delcroip) [inline conf.py:209]: why did you remove that bar ?
I used that function to inject links in the side bar

File: user_manual/claims/batch_claims.rst [hunk 2]
```diff
@@ -3,195 +3,171 @@
...
+  Category
+    Select category of health care (**In-patient/hospital, Out-patient/Non-hospital, All**) from the list of categories of health care by clicking on the arrow on the right of the selector.
 
-    * ``Show Claims``
+  Filter
+    Once criteria are chosen, clicking on this filter button (:numref:`batch_run_filter`) will filter based on the selection criteria.
 
-      Check this checkbox, if you need to show all claims in detailed way in the protocol.
+    .. _batch_run_filter:
+    .. figure:: /img/user_manual/mat.send.png
```
Comment (delcroip) [inline user_manual/claims/batch_claims.rst]: not the correct picture

**Comment by delcroip** [comment]
http://docs.openimis.org/en/update-new-modules used for the review

File: index.rst [hunk 3]
```diff
@@ -5,33 +5,32 @@
...
-    - `<Hyperlink>`_  enable a quick link (using the online version) to the subject relating to the functionality,
-    - **Item** means an item in a drop down list,
-    - ``LABEL`` means a data field or a button,
-    - _NAME_OF_PAGE_ means a name of page or a data field in a text without hyperlink.
+The open Insurance Management Information System ( openIMIS ) is a web based software to manage health insurance schemes. It includes functionality for setup of the software to requirements of health insurance schemes, administration of policies and policyholders and for claim processing.
 
-Welcome to the openIMIS technical documentation repository.
+This manual is a guide on the use and functionality of the software rather than in-depth technical reference. The Contents section, provide a reference to the page of each major chapter and the sub chapters within. By clicking on the content title (online version), the reader is re-directed to the position of the content title.
 
-.. .. include:: /_sidebar.rst.inc
```
Comment (qgerome) [inline index.rst:23]: @delcroip I removed the gensidebar.py because this has been commented. It's not included in the rst anymore. Is it used outside ? I can revert my change if needed.

File: conf.py (hunk 1 above)
Comment (qgerome) [reply conf.py:209]: As stated in another comment, I removed it because I noticed that the result of this script was not included anymore in the documentation. If it's used outside of the docs, I'll revert my change with pleasure.

Comment (delcroip) [reply conf.py:209]: If I recall it runs only if you remove the "toctree" cache

**Comment by qgerome** [comment]
Ok, in doubt I'll revert this change then.
Quentin Gérôme
+32 471 490 190

File: conf.py (hunk 1 above)
Comment (delcroip) [reply conf.py:209]: https://github.com/openimis/openimis_docs/blob/d8283a3392b6289f78bc03e8833c960a505d04c6/conf.py#L209

Comment (qgerome) [reply conf.py:209]: I've put it back.

**Review by delcroip — APPROVED** [review]
